api/src/main/resources/applicationContext-service.xml

```diff
@@ -597,6 +597,14 @@
 
 	<!--  ******************************  MESSAGE SERVICES  ****************************** -->
 
+    <bean id="messageServiceBean" class="org.openmrs.notification.MessageService">
```

Comment:
How is this bean different from the `messageService` defined below and why do we need it?